content/agent/faq/agent-commands.md

```diff
@@ -40,7 +40,7 @@ aliases:
 |Platform|Agent v5 |Agent v6|
 |:--------|:-----|:--------|
 |Linux|`sudo service datadog-agent status`|`sudo datadog-agent status`|
-|Docker|`sudo docker exec -it dd-agent /etc/init.d/datadog-agent status`|`n/a`|
+|Docker|`sudo docker exec -it dd-agent /etc/init.d/datadog-agent status`|`sudo docker exec -it datadog-agent agent status``|
```

Comment:
There's a subtle difference here, `datadog-agent status` prints the info page of the agent, not the status of the service. We should either provide a command able to show the init system status or drop a note explaining why it's not available.

Reply:
`s6-svstat /var/run/s6/services/agent/` should do the trick for the service status

Reply:
Alright, hold on, that made me realize the command for agent 5 is broken.